e2e_tests/e2e.test.ts

```diff
@@ -230,108 +299,54 @@ describe("webdriver", () => {
-//         }).finally(() => killDriver(driver))
-//     })
-
-//     test("`:tabopen example.org<CR>` opens example.org.", async () => {
-//         const driver = await getDriver()
-//         return newTabWithoutChangingOldTabs(driver, async () => {
-//             await sendKeys(driver, ":tabopen example.org<CR>")
-//         }).then(async ([newtab, _]) => {
-//             expect(newtab.active).toEqual(true)
-//             await driver.wait(untilTabUrlMatches(driver, newtab.id, "example.org"), 10000)
-//         }).finally(() => killDriver(driver))
-//     })
-
-//     test("`:tabopen search duckduckgo<CR>` opens google.", async () => {
-//         const driver = await getDriver()
-//         return newTabWithoutChangingOldTabs(driver, async () => {
-//             await sendKeys(driver, ":tabopen search duckduckgo<CR>")
-//         }).then(async ([newtab, _]) => {
-//             expect(newtab.active).toEqual(true)
-//             await driver.wait(untilTabUrlMatches(driver, newtab.id, new RegExp("^https://www.google.com/search.*duckduckgo")), 10000)
-//         }).finally(() => killDriver(driver))
-//     })
-
-//     test("`:tabopen -b about:blank<CR>` opens a background tab.", async () => {
-//         const driver = await getDriver()
-//         return newTabWithoutChangingOldTabs(driver, async () => {
-//             await sendKeys(driver, ":tabopen -b about:blank<CR>")
-//         }).then(async ([newtab, _]) => {
-//             expect(newtab.active).toEqual(false)
-//             await driver.wait(untilTabUrlMatches(driver, newtab.id, "about:blank"))
-//         }).finally(() => killDriver(driver))
-//     })
-
-//     test("`:tabopen -c work about:blank<CR>` opens about:blank in a container.", async () => {
-//         const driver = await getDriver()
-//         return newTabWithoutChangingOldTabs(driver, async () => {
-//             await sendKeys(driver, ":tabopen -c work about:blank<CR>")
-//         }).then(async ([newtab, _]) => {
-//             expect(newtab.active).toEqual(true)
-//             expect(newtab.cookieStoreId).toMatch("firefox-container-")
-//             await driver.wait(untilTabUrlMatches(driver, newtab.id, "about:blank"))
-//         }).finally(() => killDriver(driver))
-//     })
-
-//     test("`:tabopen -b -c work search qwant<CR>` opens about:blank in a container.", async () => {
-//         const driver = await getDriver()
-//         return newTabWithoutChangingOldTabs(driver, async () => {
-//             await sendKeys(driver, ":tabopen -b -c work search qwant<CR>")
-//         }).then(async ([newtab, _]) => {
-//             expect(newtab.active).toEqual(false)
-//             expect(newtab.cookieStoreId).toMatch("firefox-container-")
-//             await driver.wait(untilTabUrlMatches(driver, newtab.id, new RegExp("^https://www.google.com/search.*qwant")))
-//         }).finally(() => killDriver(driver))
```

Comment:
If the tests are less flakey now it'd be good to re-enable these three since they test quite complicated bits of Tridactyl

Reply:
Added.